draft-ietf-mls-protocol.md

```diff
@@ -1539,6 +1539,7 @@ enum {
   external(1),
   reinit(2),
   branch(3),
+  resync(4)
```

Comment:
I am not adverse to a resync option. However, this was in the original PR for PSKs, discussed by the group then removed. At that time it was called "recover" (original WG brief") and later "resumption". There was fairly extensive discussion on it, so we should probably double-check what was kept/removed and why in the mailinglist and meeting notes before re-introducing this.

Reply:
Actually the "resumption" part was kept as a separate key from the key schedule, but downgraded from use as group "recovery" and applied strictly to branch/reinit. PR #421 shows the changed text.

Reply:
I'm afraid, I can't recall the reasons either. I still think this might be useful to have, though.